src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -25,62 +25,70 @@
-                <attribute name="security">is_granted('IS_AUTHENTICATED_ANONYMOUSLY')</attribute>
+
+            <collectionOperation name="shop_post">
+                <attribute name="method">POST</attribute>
+                <attribute name="path">/admin/orders</attribute>
+                <attribute name="messenger">input</attribute>
+                <attribute name="input">Sylius\Bundle\ApiBundle\Command\Cart\PickupCart</attribute>
                 <attribute name="openapi_context">
                     <attribute name="summary">Pickup a new cart</attribute>
                 </attribute>
-                <attribute name="messenger">input</attribute>
-                <attribute name="input">Sylius\Bundle\ApiBundle\Command\Cart\PickupCart</attribute>
             </collectionOperation>
         </collectionOperations>
 
         <itemOperations>
-            <itemOperation name="get" />
-            <itemOperation name="delete">
+            <itemOperation name="admin_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/admin/orders/{id}</attribute>
+            </itemOperation>
+
+            <itemOperation name="shop_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/shop/orders/{id}</attribute>
+            </itemOperation>
+
+            <itemOperation name="shop_delete">
+                <attribute name="method">DELETE</attribute>
+                <attribute name="path">/shop/orders/{id}</attribute>
                 <attribute name="openapi_context">
                     <attribute name="summary">Delete cart</attribute>
                 </attribute>
             </itemOperation>
 
-            <itemOperation name="cancel">
-                <attribute name="security">is_granted('ROLE_API_ACCESS')</attribute>
-                <attribute name="openapi_context">
-                    <attribute name="summary">Cancels Order</attribute>
-                </attribute>
+            <itemOperation name="admin_cancel">
                 <attribute name="method">PATCH</attribute>
+                <attribute name="path">/admin/orders/{id}/cancel</attribute>
                 <attribute name="input">false</attribute>
-                <attribute name="path">/orders/{id}/cancel</attribute>
                 <attribute name="controller">sylius.api.order_state_machine_transition_applicator:cancel</attribute>
                 <attribute name="denormalization_context">
                     <attribute name="groups">order:update</attribute>
                 </attribute>
-            </itemOperation>
-
-            <itemOperation name="add_item">
-                <attribute name="security">is_granted('IS_AUTHENTICATED_ANONYMOUSLY')</attribute>
                 <attribute name="openapi_context">
-                    <attribute name="summary">Add Item to cart</attribute>
+                    <attribute name="summary">Cancels Order</attribute>
```

Comment:
here is used `cancels` that sounds like method description or action. But next You use `Add item` which sounds like we describe the action that user can make with this method. Shouldn't we use consistent naming? Here 'cancel' sounds better IMO :)

Reply:
We should be consistent for sure. In addition, let's look what ApiPlatform provides here already: 
![image](https://user-images.githubusercontent.com/6213903/92477909-acf69c00-f1e1-11ea-8db1-367604297fdc.png)
I would vote for `cancels` and `adds item` therefore.